bloom/commands/release.py

```diff
@@ -296,6 +297,16 @@ def validate_github_url(url, url_type):
     return True
 
 
+def infer_release_repo_from_env(repository):
```

Comment:
Can you please add a docstring to this function describing the environment variable's behavior.